transport-classes-io_uring/src/main/java/io/netty/channel/uring/MsgHdrMemory.java

```diff
@@ -24,6 +24,9 @@
 
 final class MsgHdrMemory {
     private static final byte[] EMPTY_SOCKADDR_STORAGE = new byte[Native.SIZEOF_SOCKADDR_STORAGE];
+    private static final ByteBuffer GLOBAL_IOV_BASE =  Buffer.allocateDirectWithNativeOrder(1);
+    private static final long GLOBAL_IOV_BASE_ADDRESS = Buffer.memoryAddress(GLOBAL_IOV_BASE);
+    private static final int GLOBAL_IOV_LEN = 1;
```

Comment:
Is it really safe to use static buffer here ? Wouldn't it be better to not use a static and why is the capacity  1 ?

Reply:
It is safe.This is only used for sendfd. 

The capacity 1 here comes from
``` c
static jint netty_unix_socket_sendFd(JNIEnv* env, jclass clazz, jint socketFd, jint fd) {
//.. 
    struct iovec iov[1] = { { 0 } };
    char iovecData[1];
//.....
        descriptorMessage.msg_iov = iov;
        descriptorMessage.msg_iovlen = 1;
        iov[0].iov_base = iovecData;
```